test/builtin/files_io/test_files.py

```diff
@@ -349,6 +348,52 @@ def test_private_doctests_files(str_expr, msgs, str_expected, fail_msg):
     )
 
 
+def test_open_read():
+    """ """
+    new_temp_file = NamedTemporaryFile(mode="r", delete=True)
+    name = canonic_filename(new_temp_file.name)
+    os.unlink(name)
+    check_evaluation(
+        str_expr=f'OpenRead["{name}"]',
+        str_expected=f"OpenRead[{name}]",
+        to_string_expr=True,
+        hold_expected=True,
+        failure_message=None,
+        expected_messages=(f"Cannot open {name}.",),
+    )
+
+
+# rocky: I don't understand what these are supposed to test.
+# (
+#     r"Hold[<<`/.\-_:$*~?] // FullForm",
```

Comment:
Here's the problem with this test.. "\" is a path separator in MS Windows. It is not a separator in POSIX systems. 

Even on POSIX systems pytest is giving a warning because it sees the string as trying to escape a character "-" which is not a valid escape character. 

What gives here? In WMA is it really true that `Hold[<<`/.\-_:$*~?]`  comes back something like `Hold[Get["`/.\\\\-_:$*~?"]]` ? 

(The server I use to check WMA expression is not currently accessible).